.github/workflows/basic.yml

```diff
@@ -42,17 +43,23 @@ jobs:
             compiler: {c: gcc, cxx: g++}
             shared_library: 'OFF'
             os_provider: 'OFF'
+        exclude:
+          # TODO: Remove this exclusion. There's a leak reported by sanitizer
+          # in jemalloc pool implementation that points to a tracker_value_t
+          # not being free'd.
```

Comment:
free'd -> freed

Reply:
I've rewritten this comment, it shouldn't be an issue now.